src/riak_kv_pb_index.erl

```diff
@@ -109,7 +107,7 @@ process(#rpbindexreq{} = Req, State) ->
                     maybe_perform_query(QueryVal, Req, State)
             end;
         false ->
-            {error, "Secondary index queries have been disabled in the configuration", State}
```

Comment:
Does it really get us much of a benefit to make all the error messages look similar? It seems to me that the most important thing in an error message is that it clearly describes what went wrong. IMO the former sentence does a better job of explaining exactly what happened than the one that replaced it here.

I'll admit I'm being kind of nit-picky, so if you disagree then you don't have to change it, but I wanted to at least revisit this decision briefly.